src/utils/smartWallet.js

```diff
@@ -54,25 +54,34 @@ const getMessage = (
       return {
         title: 'Assets are being transferred to Smart Wallet',
         message: 'You will be able to send assets once submitted transfer is complete' +
-          `${activeAccountType === ACCOUNT_TYPES.SMART_WALLET ? ' and Smart Wallet is deployed' : ''}.` +
+          `${isSmartWalletActive ? ' and Smart Wallet is deployed' : ''}.` +
           `\nCurrently ${complete} of ${total} assets are transferred.`,
       };
     default:
       return {};
   }
 };
 
-export function getSmartWalletStatus(accounts: Accounts, smartWalletState: Object): SmartWalletStatus {
-  const account = accounts.find(acc => acc.type === ACCOUNT_TYPES.SMART_WALLET);
-  const activeAccount = getActiveAccount(accounts) || {};
+export const hasSmartWalletAccount = (accounts: Accounts): boolean => {
```

Comment:
Duplicate (and not working) method. Please check for `userHasSmartWallet` method in this file :)